python/_restclient/hydrotools/_restclient/_restclient.py

```diff
@@ -1,211 +1,231 @@
-"""
-Generic REST Api client usable/inheritable for general usage and adaptable to more
-specific use cases.
+#!/usr/bin/env python3
```

Comment:
Did you mean to replace the heading docstring with a shebang for a non-executable file?

Reply:
Yes/No, i have my IDE automatically do that in python files for better or worse. Ill remove it and others that are not executable. Something I enabled a long time ago and never looked back.